views/patterns/02-organisms/global/header.blade.php

```diff
@@ -7,6 +7,8 @@
     $logo = get_alps_option('logo');
   }
 
+  $logo_nad = get_alps_option('logo_nad');
```

Comment:
Can this variable be renamed to `logo_custom`? The first instance will be NAD, but if others want to use it, having it more generic might be better, right?